src/domain/synthetics/trade/utils/withdrawal.ts

```diff
@@ -55,12 +60,17 @@ export function getWithdrawalAmounts(p: {
   }
 
   if (strategy === "byMarketToken") {
-    values.marketTokenAmount = marketTokenAmount;
-
-    if (vaultInfo) {
-      const glvPrice = vaultInfo.indexToken.prices.minPrice;
-      values.marketTokenUsd = convertToUsd(marketTokenAmount, vaultInfo.indexToken.decimals, glvPrice)!;
+    if (glvInfo) {
+      values.glvTokenAmount = glvTokenAmount!;
+      values.glvTokenUsd = convertToUsd(glvTokenAmount, glvToken?.decimals, glvToken?.prices.minPrice)!;
+      values.marketTokenAmount = convertToTokenAmount(
+        values.glvTokenUsd,
+        marketToken.decimals,
+        marketToken.prices.maxPrice
+      )!;
+      values.marketTokenUsd = marketTokenAmountToUsd(marketInfo, marketToken, values.marketTokenAmount)!;
```

Comment:
just out of curiosity, won't it be same as `values.glvTokenUsd`?

Reply:
huh, indeed, fixed